kyototycoon/README.md

```diff
@@ -54,7 +54,7 @@ The kyototycoon check is compatible with all major platforms.
 ## Data Collected
 ### Metrics
 
-See [metadata.csv](https://github.com/DataDog/integrations-core/blob/master/kyototycoon/metadata.csv) for a list of metrics provided by this information.
+The Kyototycoon check does not include any metric at this time.
```

Comment:
I don't think that's true:

https://github.com/DataDog/integrations-core/blob/master/kyototycoon/check.py#L98
https://github.com/DataDog/integrations-core/blob/master/kyototycoon/check.py#L102
https://github.com/DataDog/integrations-core/blob/master/kyototycoon/check.py#L27
...

But indeed the metadata.csv is outdated: https://github.com/DataDog/integrations-core/blob/master/kyototycoon/metadata.csv

Reply:
@hush-hush Let's keep it this way for now, and I'm going to do a PR to add metrics.